docs/modules/operation/pages/deep-dive/admin/housekeeping/introduction.adoc

```diff
@@ -80,6 +80,7 @@ Usage statistics include the following data points:
 * Core Flows persisted count
 * Core Newts samples inserted count
 * Core Queued updates completed
```

Comment:
In line with this change and the one immediately below, all entries in this list should have a period at the end of the sentence.

```suggestion
* Number of core flows persisted.
* Number of Newts samples inserted.
* Number of queued updates completed.
```